data/db-schema/src/main/resources/net/corda/db/schema/vnode-vault/migration/ledger-utxo-creation-v1.0.xml

```diff
@@ -0,0 +1,211 @@
+                                 referencedColumnNames="transaction_id,group_idx,leaf_idx"
+                                 referencedTableName="utxo_transaction_component"/>
+
+        <createTable tableName="utxo_transaction_output">
+            <column name="transaction_id" type="VARCHAR(160)">
+                <constraints nullable="false"/>
+            </column>
+            <column name="group_idx" type="INT">
+                <constraints nullable="false"/>
+            </column>
+            <column name="leaf_idx" type="INT">
+                <constraints nullable="false"/>
+            </column>
+            <column name="type" type="VARCHAR(255)">
+                <constraints nullable="false"/>
+            </column>
+            <column name="token_type" type="VARCHAR(255)">
+                <constraints nullable="true"/>
+            </column>
+            <column name="token_issuer_hash" type="VARCHAR(160)">
+                <constraints nullable="true"/>
+            </column>
+            <column name="token_notary_x500_name" type="VARCHAR(1024)">
+                <constraints nullable="true"/>
+            </column>
+            <column name="token_symbol" type="VARCHAR(255)">
+                <constraints nullable="true"/>
+            </column>
+            <column name="token_tag" type="VARCHAR(255)">
+                <constraints nullable="true"/>
+            </column>
+            <column name="token_owner_hash" type="VARCHAR(160)">
+                <constraints nullable="true"/>
+            </column>
+            <column name="token_amount" type="NUMERIC">
+                <constraints nullable="true"/>
+            </column>
+            <column name="created" type="TIMESTAMP">
+                <constraints nullable="false"/>
+            </column>
+        </createTable>
+        <addPrimaryKey columnNames="transaction_id, group_idx, leaf_idx"
+                       constraintName="utxo_transaction_output_pkey"
+                       tableName="utxo_transaction_output"/>
+        <addForeignKeyConstraint baseColumnNames="transaction_id,group_idx,leaf_idx"
+                                 baseTableName="utxo_transaction_output"
+                                 constraintName="fk_utxo_transaction_output"
+                                 referencedColumnNames="transaction_id,group_idx,leaf_idx"
+                                 referencedTableName="utxo_transaction_component"/>
+
+        <createTable tableName="utxo_transaction_relevancy">
+            <column name="transaction_id" type="VARCHAR(160)">
+                <constraints nullable="false"/>
+            </column>
+            <column name="group_idx" type="INT">
+                <constraints nullable="false"/>
+            </column>
+            <column name="leaf_idx" type="INT">
+                <constraints nullable="false"/>
+            </column>
+            <column name="relevancy" type="BOOLEAN">
```

Comment:
Should that be called `isRelevant` or `is_relevant`? A field called `relevancy` make me expect something more descriptive than a boolean.

Reply:
that was fixed